pkg/piecestore/psserver/psdb/psdb.go

```diff
@@ -63,7 +60,7 @@ func Open(ctx context.Context, storage *pstore.Storage, DBPath string) (db *DB,
 	db = &DB{
 		DB:      sqlite,
 		storage: storage,
-		check:   time.NewTicker(*defaultCheckInterval),
+		check:   time.NewTicker(time.Hour), // TODO: make configurable
```

Comment:
this was configurable? i don't understand what is motivating the removal of the global flags. what do we gain by not having the global flags?

Reply:
Anyways, removed changes in piecestore and will make a proper PR to fix them.